packages/react-scripts/scripts/lint.js

```diff
@@ -0,0 +1,29 @@
+// @remove-on-eject-begin
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+// @remove-on-eject-end
+'use strict';
+
+const spawn = require('react-dev-utils/crossSpawn');
+const args = process.argv.slice(2);
+const scriptIndex = args.indexOf('lint');
+
+const result = spawn.sync(
+  'node',
+  [require.resolve('eslint/bin/eslint')]
+    .concat(['src/**/*.{js,jsx,mjs}'])
+    .concat(args.slice(scriptIndex + 1)),
+  { stdio: 'inherit' }
+);
+if (args.indexOf('--init') >= 0) {
+  console.log(
```

Comment:
What is this for? I don't understand.

Reply:
A part of `eslint --init` wizard is to install dependencies using `npm i`. Since we have yarn, this will make a small mess. This line of code is to inform, that after `react-scripts lint` project has to be rebuilt with `rm node_modules && yarn` and possibly `rm package-lock.json`.

Reply:
I added this info after losing some time figuring out why all commands don't work anymore after `react-scripts lint`.